### PR TITLE
Add slides for lecture 2

### PR DESCRIPTION
**Pull request by alexandruradovici**
### Pull Request Overview

This pull request adds slides for lecture 2.

### Testing Strategy

This pull request was tested by using a local http server.

### TODO or Help Wanted

Feedback

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Alexandru Radovici <Yalexandru.radovici@wyliodrin.com>

### CODE REVIEW

File: slides/lectures/02/mmio/slides.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+---
+layout: two-cols
+---
+# STM32L0x2
+A real MCU
+
+| | |
+|-|-|
+| Cortex-M0+ Peripherals | MCU's *settings* and internal peripherals, available at the same address on all M0+ |
+| Peripherals | GPIO, UART, SPI, I2C, US, etc |
```
Comment (DTeodor-Alexaandru) [inline slides/lectures/02/mmio/slides.md:22]:
```suggestion
| Peripherals | GPIO, USART, SPI, I2C, USB, etc |
```

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Review by TeodoraMiu — DISMISSED** [review]

**Review by DanutAldea — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

File: slides/lectures/02/mmio/slides.md (hunk 1 above)
Comment (alexandruradovici) [reply slides/lectures/02/mmio/slides.md]: USART is specific, usually it is UART, no?

File: slides/lectures/02/rust-embedded/slides.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
+layout: section
+---
+# Rust Embedded HAL
```
[the remark this answers is not in the document]
Comment (alexandruradovici) [reply slides/lectures/02/rust-embedded/slides.md:4]: The next slides has in the stack image.

[the remark this answers is not in the document]
Comment (alexandruradovici) [reply slides/lectures/02/rust-embedded/slides.md:4]: I see what you mean, it is the STm32 example and it is like you say.

File: slides/lectures/02/mmio/slides.md (hunk 1 above)
Comment (alexandruradovici) [reply slides/lectures/02/mmio/slides.md]: I see what you mean, it is the STM32 example, and it is USART. Done.

**Review by DanutAldea — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**